validator/keymanager/imported/keymanager.go

```diff
@@ -172,6 +175,28 @@ func (opts *KeymanagerOpts) String() string {
 	return b.String()
 }
 
+// Convert KeymanagerOpts struct to a map[string]string
+func (opts *KeymanagerOpts) ConvertToConfig() map[string]string {
```

Comment:
Can we include this function somewhere instead where we need to do this conversion? Perhaps in the actual RPC package. It should be also likely be an unexported function since it doesn't have any other use